google/cloud/spanner_v1/_opentelemetry_tracing.py

```diff
@@ -56,36 +56,40 @@ def get_tracer(tracer_provider=None):
 
 
 @contextmanager
-def trace_call(name, session, extra_attributes=None, observability_options=None):
-    if session:
-        session._last_use_time = datetime.now()
-
-    if not HAS_OPENTELEMETRY_INSTALLED or not session:
+def trace_call(name, session=None, extra_attributes=None, observability_options=None):
+    if not (HAS_OPENTELEMETRY_INSTALLED and name):
         # Empty context manager. Users will have to check if the generated value is None or a span
         yield None
         return
 
+    if session:
```

Comment:
@odeke-em 
this is not yet addresses.

Reply:
I removed that @harshachinta as it is from the bigger PR and I don't think we shall need it anymore once some tests are sent in the other PR. I deleted it.

Reply:
It is not from your PR, this was added by us to handle one case. Don't remove that
I am talking about this code. We need this to know when was a session last used to avoid making a backend call to verify session existence.
```
    if session:
        session._last_use_time = datetime.now()
```